### DISCUSSION TITLE
What Feature Should we add next?

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#42] [Polls]
You read the title

**Comment by ThatOneSillyGuyDon** [comment]
Softcoded Menus are already added, just figuring out some bugs I gotta fix

**Reply by Theoyeah** [reply]
a

**Reply by RealDealJinx** [reply]
> Softcoded Menus are already added, just figuring out some bugs I gotta fix

Correction

Moving Menus With Soft Code Already Added
Adding Menus isnt possible rn
